packages/snap/src/utils/types.ts

```diff
@@ -0,0 +1,35 @@
+import * as t from 'io-ts'
+
+export const TransactionConfig = t.partial({
+    from: t.union([t.string, t.number]),
+    to: t.string,
+    value: t.union([t.number, t.string, t.bigint]),
+    gas: t.union([t.string, t.number]),
+    gasPrice: t.union([t.number, t.string, t.bigint]),
+    data: t.string,
+    nonce: t.number,
+    chainId: t.string,
+    common: t.string,
+    chain: t.string,
+    hardfork: t.string
+})
+
+export const CeloParams = t.partial({
+    feeCurrency: t.string,
+    gatewayFeeRecipient: t.string,
+    gatewayFee: t.string
```

Comment:
this is a quantity

Reply:
Please see comment https://github.com/celo-org/gas-snap/pull/17#discussion_r1238822859